starknet_py/tests/e2e/client/conftest.py

```diff
@@ -105,6 +105,11 @@ def fixture_deploy_transaction_hash():
     return 0x11C1C6731ACE34AB4A9137A82092F26ECE38E7428E5E2028DA587893AAE0E02
 
 
+@pytest.fixture(name="declare_transaction_hash")
+def fixture_declare_transaction_hash():
+    return 0x77CCBA4DF42CF0F74A8EB59A96D7880FAE371EDCA5D000CA5F9985652C8A8ED
+
+
 @pytest.fixture(name="contract_address")
```

Comment:
We'll be moving away from the hardcoded values so ideally we would replace this based on changes from [this PR](https://github.com/software-mansion/starknet.py/pull/268) before the merge.